components/base/kinematicbase/kinematics.go

```diff
@@ -18,8 +18,9 @@ import (
 const (
 	distThresholdMM         = 100
 	headingThresholdDegrees = 15
-	defaultAngularVelocity  = 60  // degrees per second
-	defaultLinearVelocity   = 300 // mm per second
+	defaultAngularVelocity  = 60    // degrees per second
+	defaultLinearVelocity   = 300   // mm per second
+	deviationThreshold      = 300.0 // mm
```

Comment:
is this distance fine for the deviationThreshold? Not sure what use cases this will be used for an how big of a tolerance there should be